hubmap/hubmap/urls.py

```diff
@@ -13,11 +13,41 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from django.conf import settings
+from django.conf.urls import url
 from django.contrib import admin
 from django.urls import include, path
+from django.views.generic import TemplateView
+from rest_framework.authtoken import views
+from rest_framework.documentation import include_docs_urls
+from rest_framework.schemas import get_schema_view
+
+from dataportal import views as dataportal_views
+
+API_TITLE = 'HuBMAP UI-backend API'
+API_DESCRIPTION = 'A Web API for viewing HuBMAP Consortium experiments data.'
+schema_view = get_schema_view(title=API_TITLE)
+
+
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/', include('dataportal.urls')),
+    path('', dataportal_views.globus, name='globus'),
     path('', include('frontend.urls')),
+    path('', include('django.contrib.auth.urls')),
+    path('loggedin/', dataportal_views.GlobusUserAuth.as_view(), name='loggedin'),
+    path(f'{settings.AUTH_URL_PREFIX}/', include('rest_framework_social_oauth2.urls')),
+    path('logout/globus/', dataportal_views.logout, name='logout'),
```

Comment:
Is the logout logic specific to Globus authentication? We clear some cookies that we set, but those aren't Globus-specific. I don't see a reason to hardcode `globus` in the URL; we can expose `dataportal.views.logout` at the `logout/` URL by moving this above line 38.